ansible_ai_connect/main/permissions.py

```diff
@@ -28,4 +29,4 @@ def has_permission(self, request, view):
             return permitted
 
         user = request.user
-        return user.rh_employee
+        return user.rh_employee or "test" in list(user.groups.values_list("name", flat=True))
```

Comment:
you can also use this expression to check group membership:
`user.groups.filter(name="test").exists()` 
which was used previously in https://github.com/ansible/ansible-ai-connect-service/pull/1341/files#diff-d8b9627bf3e9cc38433bebfcb64ea46bc5dee93c7eaa4e4fc62fe6c3fa1d4341

Reply:
Oh, that's neat. I'll update the code. Thanks!